templates/js/message-extension/README.md

```diff
@@ -1,122 +1,65 @@
-# How to use this Message Extension HelloWorld app
+# Overview of Collect Form Input and Process Data
```

Comment:
Maybe "Overview of Collect Form Input and Process Data template" ?

Reply:
Updated